batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/utils/PasswordEncoder.java

```diff
@@ -0,0 +1,39 @@
+package com.lftechnology.batch7crud.utils;
+
+import com.lftechnology.batch7crud.exception.PasswordEncoderException;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This Class encodes text password into 16 byte hash value
+ * <p>
+ * Created by kiran on 1/29/16.
+ */
+public class PasswordEncoder {
+
+  public static final Logger LOGGER = Logger.getLogger(PasswordEncoder.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "LOGGER" logger private static final. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)